app/javascript/components/supplemental_application/sections/LeaseInformatonInputs.js

```diff
@@ -52,7 +68,8 @@ const LeaseInformationInputs = ({ formApi, store }) => {
         />
         <CustomFormGrid.Select
           label='Preference Used'
-          field='preference_used' options={[]} placeholder='Select One' />
+          onChange={toggleNoPreferenceUsed(formApi)}
```

Comment:
@akegan created an onchange function so that the value assignment for `noPreferenceUsed` stays within this file :)